### ISSUE TITLE
make a demo app using this library

### ISSUE THREAD

**Issue by unsuitable001** [unsuitable001/memoryJS#2]
Make a demo web-app application to show this library's functionality.

**Comment by dolani** [comment]
I will like to take a stab on this.

**Comment by jmcdona1d** [comment]
I could probably do this over the next few days for you if you assign it. To demonstrate you basically just want to display descriptions, code and its output - sort of like the readme correct?

**Comment by unsuitable001** [comment]
> I could probably do this over the next few days for you if you assign it. To demonstrate you basically just want to display descriptions, code and its output - sort of like the readme correct?

Yes. Like that. It will be even better if you make a simple but actual app. So, user may get a nice demo :smile: 
Work on it buddy. I'm glad.

**Comment by jmcdona1d** [comment]
Right, ok so add like a layer of interactivity so it's not just reading stuff?

**Comment by unsuitable001** [comment]
> Right, ok so add like a layer of interactivity so it's not just reading stuff?

Yes exactly
